plutus-ledger-api/test-plugin/Spec/golden/9.2/Long.stat.golden

```diff
@@ -1,15 +1,15 @@
-({cpu: 9780362526 | mem: 35002468})
-({cpu: 11535083020 | mem: 41619510})
-({cpu: 7773631953 | mem: 28259432})
-({cpu: 5769243585 | mem: 20971834})
-({cpu: 6924442130 | mem: 24754988})
-({cpu: 7745425322 | mem: 27984642})
-({cpu: 4446697967 | mem: 16138868})
-({cpu: 4033846923 | mem: 14573606})
-({cpu: 6808761202 | mem: 24376412})
-({cpu: 7477360785 | mem: 27205524})
-({cpu: 3869670543 | mem: 14302732})
-({cpu: 2884402957 | mem: 10629710})
-({cpu: 3952840806 | mem: 14128932})
-({cpu: 3754703216 | mem: 13744010})
-({cpu: 927859238 | mem: 3396606})
\ No newline at end of file
+({cpu: 3012083773 | mem: 11175942})
+({cpu: 713195980 | mem: 3098404})
```

Comment:
Note that those tests were written with the assumption that duplicated `CurrencySymbol`s/`TokenName`s were allowed, hence they're pretty useless for an algo that quits early in certain cases when we have duplicated names. Fixing that is a future work unrelated to this PR.